packages/common/src/util.js

```diff
@@ -631,3 +631,66 @@ function getUserAgentFromPackageJson(packageJson) {
 }
 
 util.getUserAgentFromPackageJson = getUserAgentFromPackageJson;
+
+/**
+ * Promisifies certain Class methods. This will not promisify private or
+ * streaming methods. By default it will promisify any method that is
+ * camel cased with the exception of `query`.
```

Comment:
I think this bit on `query` is no longer accurate, unless I'm missing something in the code.